requirements.md

```diff
@@ -24,36 +24,29 @@
 
 ## Use Cases
 
-1. Use Case Name (Should match functional requirement name)
-- **Pre-condition:** <can be a list or short description> Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua.
+1. User home page (user can see messages of users they follow)
```

Comment:
The use case name should be in "verb object" format. Eg: Make post

Reply:
We might have to use this same format in our list of 12 functional requirements too.